tests/ssl-opt.sh

```diff
@@ -215,6 +215,8 @@ get_config_value_or_default() {
 
 requires_config_value_at_least() {
     VAL="$( get_config_value_or_default "$1" )"
+    # Pick only the relevant information from the output
```

Comment:
Again, instead of changing the output here in the test script, we should make sure that the program produces the correct output.